gammapy/catalog/tests/test_others.py

```diff
@@ -28,6 +28,29 @@ def test_load_catalog_green(tmpdir):
 
 @pytest.mark.xfail
 @requires_data('gammapy-extra')
+def test_load_catalog_snrcat(tmpdir):
```

Comment:
I guess, I accidentially put this back when doing rebase master. So I will remove it in the next commit